panel/io/jupyter_executor.py

```diff
@@ -77,7 +77,8 @@ def __init__(self, path, token, root_url):
         try:
             self.session = self._create_server_session()
             self.connection = ServerConnection(self.protocol, self, None, self.session)
-        except Exception:
+        except Exception as e:
+            self.exception = e
```

Comment:
Nit: I would have `str(e)´ here to not have the error object saved.

Reply:
Agreed, doesn't matter a ton in this case because if there's an exception the kernel this is in will get terminated pretty much immediately.

Reply:
Yep, that is why it is a nit.